src/containers/CallIn/ThankYou/ThankYou.js

```diff
@@ -168,9 +164,11 @@ export class ThankYou extends Component {
                     overallStats: overall
                 })
             }).catch((error) => {
-                this.setState({
-                    statsError: error
-                })
+                Sentry.addBreadcrumb({
```

Comment:
Send a breadcrumb indicating that there was a stats error. This breadcrumb is not in the all in one breadcrumb pr